src/random.cpp

```diff
@@ -134,6 +134,25 @@ static uint64_t GetRdRand() noexcept
         if (ok) break;
     }
     return r1;
+#elif defined(_MSC_VER) && defined(_M_X64)
+    uint64_t r1 = 0; // See above why we initialize to 0.
+    for (int i = 0; i < 10; ++i) {
+        uint8_t ok = _rdrand64_step(&r1);
+        if (ok) break;
+    }
+    return r1;
+#elif defined(_MSC_VER) && defined(_M_IX86)
```

Comment:
Do we really care about x86 32 bit here? We don't build binaries for it anymore. I'm also not sure if there are actually existing x86 32-bit CPUs that support these instructions.

Reply:
I have 32-bit x86 support only because bitcoin currently supports 32-bit x86 rd instructions and I am just extending what already exists to the MSVC compiler. If it is decided that bitcoin-core will no longer target 32-bit x86 than it seems reasonable to remove.

Reply:
I can put together a PR which removes 32-bit rdrand and rdseed support if that is something you think would be mergeable.